src/Billing/ClearingHouse.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ *  @package   OpenEMR
+ *  @link      http://www.open-emr.org
+ *  @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c )2020. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ *
+ */
+
+namespace OpenEMR\Billing;
+
+use OpenEMR\Common\Logging\EventAuditLogger;
+use phpseclib\Net\SFTP;
```

Comment:
pretty cool @juggernautsei , wouldn't you want to add this to `.composer.json`?

Reply:
I assumed it was already there
        "phpseclib/phpseclib": "2.0.29", https://github.com/openemr/openemr/blob/1b975e59bfd14fab922490c4d43bee5adc29120f/composer.json#L48

Reply:
yep, we got it already :)

Reply:
nice, just looks weird uncapitalized